server.js

```diff
@@ -7967,6 +7967,86 @@ cache(function (data, match, sendBadge, request) {
   });
 }));
 
+// Launchpad integration
+camp.route(/^\/launchpad\/ppa\/([^/]+)\/([^/]+)\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
I suggest to replace first group `([^/]+)` with values which are currently expected there `(version|downloads)`.